server/repositories/offender.js

```diff
@@ -75,17 +80,17 @@ function offenderRepository(prisonApiHttpClient, incentivesApiHttpClient) {
   }
 
   function getAdjudicationsFor(offenderNo) {
-    return prisonApiHttpClient.get(
-      `${prisonApiBaseUrl}/offenders/${offenderNo}/adjudications`,
+    return adjudicationsApiHttpClient.get(
+      `${adjudicationsApiBaseUrl}/adjudications/${offenderNo}/adjudications`,
```

Comment:
[Here is this endpoint](https://github.com/ministryofjustice/hmpps-manage-adjudications-api/blob/9e4fb75fe356845eda7c96c66aab998f792aecee/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/controllers/AdjudicationSummaryController.kt#L142-L208) in the AdjudicationsAPI

We may need to add the `VIEW_ADJUDICATIONS` role to the client credentials as the [AdjudicationSummaryController has it configured](https://github.com/ministryofjustice/hmpps-manage-adjudications-api/blob/9e4fb75fe356845eda7c96c66aab998f792aecee/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/controllers/AdjudicationSummaryController.kt#L29), though it may have it already because it's [one of the valid roles for the existing PrisonAPI endpoint](https://github.com/ministryofjustice/prison-api/blob/a46928fa96e554e59b56723c689e6df79031e632/src/main/java/uk/gov/justice/hmpps/prison/api/resource/OffenderResource.java#L384-L385)

Reply:
See [Jira comment](https://dsdmoj.atlassian.net/browse/NN-5363?focusedCommentId=317904) for roles which have been updated